chaos/tests/reboot_test.go

```diff
@@ -0,0 +1,57 @@
+package tests
+
+import (
+	"testing"
+
+	"time"
+
+	"github.com/coreos/ktestutil/chaos"
+)
+
+func TestClusterReboot(t *testing.T) {
+	cluster, err := chaos.NewCluster(client,
+		chaos.WithMaxDisruption(1),
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := cluster.RebootAll(); err != nil {
```

Comment:
probably set a 1*minute rebootDuration or so. and make sure only 1 node is unKnown during that 1 minute.